docs/UserGuide.md

```diff
@@ -360,371 +361,371 @@ edit /c 2 /n cs2040s
+* `edit /s 1 /c CS2103T /n Bob`: Edits the name of the first student in the class CS2103T to "Bob".
+* `edit /s 2 /c CS2103T /id A0231234U`: Edits the id of the second student in the class "CS2103T" to "A0231234U".
 
 **Sample Usage:**
 
-1. Assuming you want to set the number of lessons conducted for the class CS2103T to 5.
-2. Enter the following command:
-   ```
-   setlesson /c CS2103T /l 5
-   ```
+1. Assuming you want to edit the id of the second student to "A010193Z", who is from "CS2103T" class, and the class index in the displayed list is "1".
+2. Enter the following commands:
+```
+view /c 1
+edit /s 2 /c CS2103T /id A010193Z
+```
 3. The result box will display the following message:
-   ```
-   Successfully set the number of lessons in CS2103T to 5!
-   ```
-4. You have successfully set the total number of lessons in CS2103T to 5. :tada:
+```
+Edited Student: John; Id: A010193Z; Memo:
+```
+4. You have successfully edited your student's details. :tada:
 
 <box type="info">
 
 **Information:**
 
-- If one of your student's number of lessons attended <u>exceeds</u> the NUMBER_OF_LESSONS, it is automatically reduced to NUMBER_OF_LESSONS.
-    - Example: John's current attendance in CS2103T is 5/7. <br> If you enter the command `setlesson /c CS2103T /l 3`, it will adjust John's attendance to 3/3.
+- Edits the student at the specified `STUDENT_INDEX` in the specified `CLASS_NAME`.
+- At least one of the optional fields must be provided.
+- Existing values will be updated to the input values. This means you can use the command to remove a student's memo or student's participation!
 
 </box>
 
-<box type="tip">
+<box type="warning">
 
-**Tip:**
-- This is useful when you accidentally [start a lesson](#starting-a-lesson-startlesson-c) and want to decrease the number of lesson(s) by 1.
+**Caution:**
+
+- Only valid ids will be recognised. It has to start with "A", followed by only numerical characters (0-9) and end with a single capitalised alphabet (A-Z).
+- After editing the student, the `UI` will update to show you the student list of the class that the edited student is in.
 
 </box>
 
 <br>
 
-Return to [Table of Contents](#table-of-contents)
-
-## Student commands :mortar_board:
-
-### Adding a student : `add /s`
+### Marking a student present : `mark /s`
 
-Adds a new student to your existing class.
+Marks a student present for your current lesson.
 
-**Command Format:** `add /s STUDENT_NAME /c CLASS_INDEX`
+**Command Format:** `mark /s STUDENT_INDEX /c CLASS_NAME`
 
 **Examples:**
-* `add /s John /c 1`: Adds a new student named "John" into the first class on the displayed class list.
-* `add /s Bob /c 2`: Adds a new student named "Bob" into the second class on the displayed class list.
+* `mark /s 2 /c CS2103T` : Marks the second student in your class "CS2103T" as present.
+* `mark /s 3 /c CS2102` : Marks the third student in your class "CS2102" as present.
 
 **Sample Usage:**
```

Comment:
I think you can add screenshots for this feature

Reply:
Will do so! Thanks for spotting it